packages/contracts/utils/helpers.ts

```diff
@@ -1,130 +1,115 @@
+    statSync(aragonDeploymentsInfoFilePath).size !== 0
   ) {
-    pluginInfo = JSON.parse(readFileSync(pluginInfoFilePath, 'utf-8'));
+    aragonDeploymentsInfo = JSON.parse(
+      readFileSync(aragonDeploymentsInfoFilePath, 'utf-8')
+    );
 
-    if (!pluginInfo[networkName]) {
-      pluginInfo[networkName] = {};
+    if (!aragonDeploymentsInfo[networkName]) {
+      aragonDeploymentsInfo[networkName] = {};
     }
   } else {
-    pluginInfo[networkName] = {};
+    aragonDeploymentsInfo[networkName] = {};
   }
-  return pluginInfo;
+  return aragonDeploymentsInfo;
 }
 
-function storePluginInfo(networkName: string, pluginInfo: any) {
+export function copyAragonDeploymentsInfoFromProdToLocal(
+  hre: HardhatRuntimeEnvironment
+) {
+  const productionNetworkName = getProductionNetworkName(hre);
+
+  const objToStore: any = {};
+  objToStore[hre.network.name] = getAragonDeploymentsInfo(
+    productionNetworkName
+  )[productionNetworkName];
+
+  writeFileSync(
+    '../../local-network-deployments.json',
+    JSON.stringify(objToStore, null, 2) + '\n'
+  );
+}
+
+function storePluginInfo(networkName: string, aragonDeploymentsInfo: any) {
   if (['localhost', 'hardhat', 'coverage'].includes(networkName)) {
     writeFileSync(
-      '../../plugin-info-testing.json',
-      JSON.stringify(pluginInfo, null, 2) + '\n'
+      '../../local-network-deployments.json',
+      JSON.stringify(aragonDeploymentsInfo, null, 2) + '\n'
     );
   } else {
     writeFileSync(
-      '../../plugin-info.json',
-      JSON.stringify(pluginInfo, null, 2) + '\n'
+      '../../production-network-deployments.json',
+      JSON.stringify(aragonDeploymentsInfo, null, 2) + '\n'
     );
   }
 }
 
+export function clearPluginInfo(networkName: string) {
+  if (['localhost', 'hardhat', 'coverage'].includes(networkName)) {
+    writeFileSync(
+      '../../local-network-deployments.json',
+      JSON.stringify({}, null, 2) + '\n'
+    );
+  } else {
+    throw "Information will be lost. You must delete 'production-network-deployments.json' manually.";
+  }
+}
+
 export function addDeployedRepo(
   networkName: string,
   repoName: string,
   contractAddr: string,
-  args: [],
+  args: any[],
   blockNumber: number
 ) {
-  const pluginInfo = getPluginInfo(networkName);
+  const aragonDeploymentsInfo = getAragonDeploymentsInfo(networkName);
 
-  pluginInfo[networkName]['repo'] = repoName;
-  pluginInfo[networkName]['address'] = contractAddr;
-  pluginInfo[networkName]['args'] = args;
-  pluginInfo[networkName]['blockNumberOfDeployment'] = blockNumber;
+  aragonDeploymentsInfo[networkName]['repo'] = repoName;
+  aragonDeploymentsInfo[networkName]['address'] = contractAddr;
+  aragonDeploymentsInfo[networkName]['args'] = args;
+  aragonDeploymentsInfo[networkName]['blockNumberOfDeployment'] = blockNumber;
 
-  storePluginInfo(networkName, pluginInfo);
+  storePluginInfo(networkName, aragonDeploymentsInfo);
 }
 
 export function addCreatedVersion(
```

Comment:
add where? perhaps some natspec and dev commnets would be nice to include here

Reply:
These functions got removed by now